### ISSUE TITLE
Accessiblity: Skip to navigation link broken and no tab stop to this functionality

### ISSUE THREAD

**Issue by Dawnzar** [CityofOttawa/Ottawa-ckan#23]
WCAG 2.0: Compliance violation level (A) 
1. Go to: data.ottawa.ca
2. Skip to navigation link in the header is present but the anchor for the link does not exist. 
3. There is no tab stop to this functionality.

Resolution: Apply link and add tab stop to 'Skip to content'.
![skip to content missing anchor for link](https://cloud.githubusercontent.com/assets/5271253/8010672/193de876-0b80-11e5-85f6-8bb65bcec9d5.JPG)
